ansible/configs/ansible-tower-advanced/files/ansible-tower-advanced_sample_vars.yml

```diff
@@ -0,0 +1,168 @@
+# TIP: CHANGEME needs to be adapted once for each developer, CHANGEME-ec2 needs to be adapted for each AWS sandbox
```

Comment:
The whole file seems a duplicate of the sample_vars.yml file.

Reply:
Never mind, just tried a cleaner vars sample file. Feel free to replace. ;)